### PR TITLE
Implementing scheduler in TechNotify project.

### PR DESCRIPTION
**Pull request by Arundhathy-k**
Implemented schedular in TechNotify project.
Application runs 8 hours per once and data will add or update into db automatically.

### CODE REVIEW

File: newsService/src/main/java/com/kovan/app/service/NewsService.java [hunk 1]
```diff
@@ -51,23 +58,31 @@ public NewsService(RestTemplate restTemplate, ObjectMapper objectMapper,
         this.objectMapper = objectMapper;
         this.newsRepositoryService = newsRepositoryService;
     }
+
+    @Scheduled(cron = "0 0 */8 * * *")
```
Comment (dhanyesh) [inline newsService/src/main/java/com/kovan/app/service/NewsService.java:66]: Add comments and try to explain the functionality

File: mongo-dao/src/main/resourses/application.properties [hunk 2]
```diff
@@ -1,2 +1 @@
-spring.data.mongodb.uri= mongodb+srv://Arundhathy:Ammu2002@kovan.us1gd.mongodb.net/?retryWrites=true&w=majority&appName=Kovan
-spring.data.mongodb.database = NewsDB
+spring.data.mongodb.uri= mongodb://localhost:27017/NewsDB
```
Comment (dhanyesh) [inline mongo-dao/src/main/resourses/application.properties]: why do we need this property file here ?

File: newsService/src/test/java/com/kovan/NewsServiceIT.java [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 @ExtendWith(SpringExtension.class)
 @SpringBootTest(webEnvironment = RANDOM_PORT)
-public class NewsServiceIT {
+class NewsServiceIT {
```
Comment (dhanyesh) [inline newsService/src/test/java/com/kovan/NewsServiceIT.java:17]: you have modified Test class so please attach screen shot for all testcases.

File: newsService/src/test/java/com/kovan/NewsServiceTest.java [hunk 4]
```diff
@@ -215,7 +215,7 @@ void testUpdateNewsInDb_ShouldUpdateExistingNews() throws Exception {
         when(restTemplate.getForObject(anyString(), eq(String.class))).thenReturn(apiResponse);
         when(objectMapper.readValue(anyString(), eq(NewsDto.class))).thenReturn(apiNewsDto);
         when(newsRepositoryService.saveNewsInDb(any(NewsDto.class))).thenReturn(existingNewsDto);
-        when(newsRepositoryService.findNewsInDb(eq(publishedAtDate))).thenReturn(of(existingNewsDto));
+        when(newsRepositoryService.findNewsInDb((publishedAtDate))).thenReturn(of(existingNewsDto));
```
Comment (dhanyesh) [inline newsService/src/test/java/com/kovan/NewsServiceTest.java:218]: Here as well attach the screen shot

**Review by dhanyesh — CHANGES_REQUESTED** [review]

File: mongo-dao/src/main/resourses/application.properties (hunk 2 above)
Comment (Arundhathy-k) [reply mongo-dao/src/main/resourses/application.properties]: property file removed

File: newsService/src/test/java/com/kovan/NewsServiceIT.java (hunk 3 above)
Comment (Arundhathy-k) [reply newsService/src/test/java/com/kovan/NewsServiceIT.java:17]: Screenshot attached.

File: newsService/src/test/java/com/kovan/NewsServiceTest.java (hunk 4 above)
Comment (Arundhathy-k) [reply newsService/src/test/java/com/kovan/NewsServiceTest.java:218]: Screenshot attached.

Comment (Arundhathy-k) [reply newsService/src/test/java/com/kovan/NewsServiceTest.java:218]: image

File: newsService/src/test/java/com/kovan/NewsServiceIT.java (hunk 3 above)
Comment (Arundhathy-k) [reply newsService/src/test/java/com/kovan/NewsServiceIT.java:17]: image

File: newsService/src/test/java/com/kovan/NewsServiceTest.java (hunk 4 above)
Comment (Arundhathy-k) [reply newsService/src/test/java/com/kovan/NewsServiceTest.java:218]: image

File: newsService/src/main/java/com/kovan/app/service/NewsService.java (hunk 1 above)
Comment (Arundhathy-k) [reply newsService/src/main/java/com/kovan/app/service/NewsService.java:66]: Added comments.

**Review by dhanyesh — APPROVED** [review]

**Pull request merged**